### PR TITLE
feat(select): Eva style

### PR DESCRIPTION
**Pull request by yggg**
### Please read and mark the following check list before creating a pull request:

 - [x] I read and followed the [CONTRIBUTING.md](https://github.com/akveo/nebular/blob/master/CONTRIBUTING.md) guide.
 - [x] I read and followed the [New Feature Checklist](https://github.com/akveo/nebular/blob/master/DEV_DOCS.md#new-feature-checklist) guide.
 
 #### Short description of what this resolves:

BREAKING CHANGE:

Arrow selector and element changed to svg icon.

NbSelectComponent 'isOpened' getter renamed to 'isOpen'.

NbOptionComponent and NbOptionGroupComponent 'disabledClass' getter
renamed to 'disabledAttribute'.

Following theme properties were renamed:
select-border-width -> select-[appearance]-border-width
select-max-height -> select-options-list-max-height
select-bg -> select-[appearance]-background-color
select-option-disabled-bg -> select-option-[appearance]-disabled-background-color
select-option-padding -> select-option-[appearance]-[size]-padding

Following theme properties removed:
select-checkmark-color
select-checkbox-color
select-option-disabled-opacity

### CODE REVIEW

**Comment by nnixaa** [comment]
image
seems like the smaller one isn't really smaller?

**Comment by nnixaa** [comment]
and also this is kinda strange

File: src/framework/theme/components/select/_select.component.theme.scss [hunk 1]
```diff
@@ -108,7 +108,7 @@
     }
   }
 
-  @include select-outline;
+  @include select-outline();
   @include select-filled;
```
Comment (nnixaa) [inline src/framework/theme/components/select/_select.component.theme.scss]: well, how about the others?)

**Comment by nnixaa** [comment]
And can we also fix the circular dependency warning as part of it?

**Comment by yggg** [comment]
> And can we also fix the circular dependency warning as part of it?

Will fix in the keyboard pr

File: src/framework/theme/components/select/_select.component.theme.scss (hunk 1 above)
Comment (yggg) [reply src/framework/theme/components/select/_select.component.theme.scss]: Fixed f20a192

**Pull request merged**